repository-data/application/src/main/resources/hcm-config/namespaces/hee/ctaCard.yaml

```diff
@@ -0,0 +1,115 @@
+definitions:
```

Comment:
- Could we change `title` as a string field with TextBox editor (in order to be inline with other cards/blocks) ?
- Could we rename `ctaText` as `desription` or `summary` (in order to avoid confusion around name with CTA Link Link `text` field) and amend it as a TextArea field (rather than a CKEditor in order to be inline with other cards/blocks) ? Also, could we make it as an optional field (as it is not required per figma) ?

Reply:
- Manifesto-Digital/hee-prototypes#15 is closed for now, as the prototype html for card component with an embedded button works fine.
- `title` renamed as string field, `ctaText` renamed as description and made it optional.

commited on d5a8bce.